### DISCUSSION TITLE
A question about faststore.

### DISCUSSION THREAD

**Discussion by tulindor** [vtex/faststore#1441] [Q&A] [answered]
Hi, I work for a big vtex partner and I'am trying to push a project like this to start working stores on front.

We need to know if the faststore proyect is advanced enough to mantain and develop a big store without complications, and if we can use the tools and components that we have on vtex IO by default. Even if they are similar, but enough to offer the same components that we use now to the clients.

I've been following the proyect and iam entusiastic enough to start using it and  give it a try, because this is the right developer aproach to a front end store.

**Comment by viniciusmlira** [comment, marked as answer]
hi, @tulindor!

FastStore was built to support both small business and large enterprise customers. It has powerful customization tools and our mantra is to build a platform "easy to maintain", specially in the long-term.

Regarding IO compatibility, we need to highlight that FastStore is a different product, using different technologies. Some components exist both in IO and FastStore, but they were built entirely different. 

We're working in documentations to help customers/partners to rewrite/migrate their storefronts from IO to FastStore following our recommended practices, but it's still a work in progress with no clear timeline to be finished.

We can't support you right now, if your intent to is go-live with a specific storefront. But if the goal is just to understand, browse and explore FastStore, as you work in a VTEX Partner, you can ask an experimental environment to your main interface with VTEX (a partnership manager, or similar). So we can provide a proper environment where you can experiment the product and deliver some feedback to VTEX.

If you have anything else, just let us know!

best,

@FastStore team

**Reply by tulindor** [reply]
Thanks a lot! we are following the progress of this proyect, we are intrested in use it.
